flag/service/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,18 @@
+package kubernetes
+
+import (
+	"github.com/giantswarm/operatorkit/v8/pkg/flag/service/kubernetes/tls"
+	"github.com/giantswarm/operatorkit/v8/pkg/flag/service/kubernetes/watch"
+)
+
+// Kubernetes is a data structure to hold Kubernetes specific command line
+// configuration flags.
+type Kubernetes struct {
+	Address            string
+	DisableClientCache string
+	InCluster          string
+	KubeConfig         string
+	KubeConfigPath     string
+	TLS                tls.TLS
+	Watch              watch.Watch
+}
```

Comment:
this file was previously imported from `operatorkit` repo but since we need the extra property `DisableClientCache string` i redeclared it here

Reply:
I would normally ask to make the change in operatorkit, but since operatorkit is not developed and won't be used anymore soon I guess it's ok

Reply:
That was my thinking as well